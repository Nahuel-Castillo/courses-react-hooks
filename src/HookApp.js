
import React from 'react';

export const HookApp = () => {
    return (
        <div>
            <h2> HookApp </h2>
        </div>
    )
}
