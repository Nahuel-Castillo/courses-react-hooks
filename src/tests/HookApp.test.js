import React from 'react';
// import { renderHook } from '@testing-library/react-hooks';
import { shallow } from 'enzyme';
import { HookApp } from '../HookApp';

describe('HookApp test', () => {
    
    test('Should show HookApp component correctly', () => {
        const wrapper = shallow(<HookApp />);
        expect(wrapper).toMatchSnapshot();
    });

});