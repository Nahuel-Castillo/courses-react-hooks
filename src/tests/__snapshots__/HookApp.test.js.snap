// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HookApp test Should show HookApp component correctly 1`] = `
<div>
  <h2>
     HookApp 
  </h2>
</div>
`;
